ceph/base/storagecluster.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: ocs.openshift.io/v1
+kind: StorageCluster
+metadata:
+  name: ocs-storagecluster
+  namespace: openshift-storage
```

Comment:
@tumido Can we move the `namespace` to the kustomization please?

Reply:
or just remove this, it is already specified in the kustomization

Reply:
good catch :+1: